webv2/web.go

```diff
@@ -2179,15 +2355,21 @@ type typeIssue struct {
 	DisplayT string
 }
 
+type ResourceDetails struct {
+	JobName string `json:"JobName"`
+	JobUrl  string `json:"JobUrl"`
+}
 type GeneratedResources struct {
-	DatabaseName      string
-	DatabaseUrl       string
-	BucketName        string
-	BucketUrl         string
-	DataStreamJobName string
-	DataStreamJobUrl  string
-	DataflowJobName   string
-	DataflowJobUrl    string
+	DatabaseName         string                     `json:"DatabaseName"`
+	DatabaseUrl          string                     `json:"DatabaseUrl"`
+	BucketName           string                     `json:"BucketName"`
+	BucketUrl            string                     `json:"BucketUrl"`
+	DataStreamJobName    string                     `json:"DataStreamJobName"`
+	DataStreamJobUrl     string                     `json:"DataStreamJobUrl"`
+	DataflowJobName      string                     `json:"DataflowJobName"`
+	DataflowJobUrl       string                     `json:"DataflowJobUrl"`
```

Comment:
add a comment that all the above are relevant only for the single shard flow

Reply:
Done